sdmetrics/reports/multi_table/_properties/base.py

```diff
@@ -19,6 +19,23 @@ def __init__(self):
         self._properties = {}
         self.is_computed = False
 
+    def _augment_error_msg(self, table_name):
```

Comment:
Adding this to the BaseClass is not ideal, but due to the way we aggregate errors, it is the easiest way to implement it.